### PR TITLE
Travis CI build files

### PR DESCRIPTION
**Pull request by avinash-palleti**
Adding an autosync job file which keeps track of changes in upstream repos
and create an auto commit to update submodule SHA in repo.

Signed-off-by: Avinash Reddy Palleti <avinash.reddy.palleti@intel.com>

### CODE REVIEW

**Comment by edersondisouza** [comment]
Looks interesting - but when/where is this thing run? In github itself? Some server that you control?

**Comment by avinash-palleti** [comment]
It will run on Jenkins Slave node, cadence depends on what we decide.

**Comment by edersondisouza** [comment]
Hmmm - I'm always wary of adding stuff that belongs to some external environment to repo. I mean, we use github and git, so things related to it are usually ok (like .gitignore). Imagine someone (TM) wanted to use avb-sh and also has a CI system, would this file be somehow in conflict with their own CI?

**Comment by avinash-palleti** [comment]
Usually this is the way, people who use travis they add .travis.yaml file into repository. And these jenkins job files will not conflict with any other until someone try to keep same name for jobfile (Jenkinsfile or autosync.groovy).
We can have one hidden directory to store all these files ( .ci ).

**Comment by edersondisouza** [comment]
Ok, go for it!

File: travis.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+cd dlt-daemon
+mkdir build
+cd build
+cmake .. -DCMAKE_INSTALL_LIBDIR=lib -DWITH_DLT_CXX11_EXT=ON
+make && sudo make install
+
+cd $ROOT_DIR
+git clone git://git.alsa-project.org/alsa-lib.git
+cd alsa-lib
+git checkout v1.1.6
```
Comment (keerockl) [inline travis.sh]: we support v1.1.7 now. can you update to v1.1.7?

**Comment by avinash-palleti** [comment]
@edersondisouza @keerockl Can you please review and merge this.

File: travis.sh (hunk 1 above)
Comment (keerockl) [reply travis.sh]: Please update alsalib to v1.1.7

**Review by edersondisouza — APPROVED** [review]
Looks ok.

File: travis.sh (hunk 1 above)
Comment (avinash-palleti) [reply travis.sh]: Sure. Will submit new patch with an update.

**Pull request merged**